modules/statistics/AvgRTPStatsReporter.js

```diff
@@ -695,6 +762,102 @@ export default class AvgRTPStatsReporter {
     }
 
     /**
+     * Calculates average number of pixels for the report
+     *
+     * @param {map} peerResolutions a map of peer resolutions
+     * @param {boolean} isLocal if the average is to be calculated for the local
+     * video or <tt>false</tt> if for remote videos.
+     * @param {VideoType} videoType
+     * @return {number|NaN} average number of pixels or <tt>NaN</tt> if there
+     * are no samples.
+     * @private
+     */
+    _calculateAvgVideoPixels(peerResolutions, isLocal, videoType) {
```

Comment:
so this method is basically just filtering the set of resolutions to process, right?  it's the other method that is doing the actual averaging?

also, why do we even need the filtering here?  every time we call it for remote, we also call it for local...so why not just do the processing all at once?